lib/Cargo.toml

```diff
@@ -16,7 +16,7 @@ path = "uniffi-bindgen.rs"
 required-features = ["uniffi/cli"]
 
 [dependencies]
-yrs = "0.17.4"
+yrs = "0.18.2"
```

Comment:
The rest of the PR is just trying to fix things that break after this version change